### ISSUE TITLE
levels has to be activated each time I open a file

### ISSUE THREAD

**Issue by mattsawyer77** [mazurov/sublime-levels#9]
Not sure if this is the intended behavior, but it's a little annoying to have to activate the plugin all the time. BTW I'm running ST3, and I don't see any errors in the console when using the plugin.

**Comment by mazurov** [comment]
Yes, agree. It should be an option to enable plugin for all js files, not only for selected one.
